sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -82,6 +95,30 @@ export interface EventDataBatch {
     tryAdd(eventData: EventData | AmqpAnnotatedMessage, options?: TryAddOptions): boolean;
 }
 
+// @public
+export class EventHubBufferedProducerClient {
```

Comment:
changelog @deyaaeldeen ?

Reply:
@ramya-rao-a was kind enough to add the entry before, see the [`main` version](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/eventhub/event-hubs/CHANGELOG.md). I am updating the release date in https://github.com/Azure/azure-sdk-for-js/pull/18672.

Reply:
:)